bucket/prismlauncher-qt5.json

```diff
@@ -3,12 +3,27 @@
     "description": "An open source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-6.3.zip",
-    "hash": "003b54560935c341ab825d41962323e56d649ea15a29f69d4ce64b93326c866c",
+    "notes": [
+        "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+        "This package is now using the portable version of Prism Launcher, and data should have been migrated automatically. If you are using a global install, you will need to copy your own data to the new Scoop persist directory"
+    ],
+    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.3/PrismLauncher-Windows-MSVC-Legacy-Portable-6.3.zip",
+    "hash": "521804443e78a68cdc62f6f1645756f8d74a15ab1941c7ff48a7deb37d419ee3",
     "suggest": {
         "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
     },
+    "pre_install": [
+        "$appdataPath = \"$HOME\\AppData\\Roaming\\PrismLauncher\"",
+        "if ( (Test-Path -Path $appdataPath\\*) -and (!(Test-Path -Path $persist_dir\\*)) -and (!($global)) ) {",
```

Comment:
Instead of completely blocking migration for global installs, I think that it's best to use implement either:
- A check to see if there's one user account in Windows, if so, migrate
- Recursively check the folders of `C:\Users` to see if there's more than one `\AppData\Roaming\PrismLauncher` folder, if there's only one found, migrate
  - Permissions could be an issue though, not sure if users can read other user data easily like that by default

However, if it's determined that it can't auto-migrate, then a warning should be displayed:
```powershell
Write-Warning "A global Scoop installation was detected with multiple user accounts. Therefore, it can't be determined which account to automatically migrate Prism Launcher data from, so you will need to manually migrate your Prism Launcher data from AppData."
```

Reply:
i don't think either of these are really a good a idea. it isn't very hard to copy things from appdata manually, and both of these add a lot of unneeded complication (to an already somewhat complicated preinstall script)

Reply:
this is actually really good, i'll add this now :)